src/EnergyPlus/OutputReportTabular.hh

```diff
@@ -213,6 +213,9 @@ namespace OutputReportTabular {
     extern bool displayEconomicResultSummary;
     extern bool displayHeatEmissionsSummary;
     extern bool displayEioSummary;
+    extern bool displayThermalResilienceSummary;
```

Comment:
Are these reset in any clear_state calls?

Reply:
@Myoldmopar I suppose they are reset [here](https://github.com/NREL/EnergyPlus/blob/9805e969689297e1395bc106d9cc9ac4d1cbf0c5/src/EnergyPlus/OutputReportTabular.cc#L594).

Reply:
Got it, I zipped right past that.  OK, the problem is elsewhere.